Library/Homebrew/shims/super/cc

```diff
@@ -216,8 +212,8 @@ class Cmd
       # used for -Xpreprocessor -fopenmp
       args << arg << enum.next
     when /-mmacosx-version-min=(\d+)\.(\d+)/
-      if high_sierra_or_later? && Regexp.last_match(1) == "10" && Regexp.last_match(2).to_i < 9
-        arg = "-mmacosx-version-min=10.9"
+      if Regexp.last_match(1) == "10" && Regexp.last_match(2).to_i < 15
+        arg = "-mmacosx-version-min=10.15"
       end
       args << arg
```

Comment:
The bump from 10.9 to 10.15 is a risky change. The code here is solely because <10.9 returns a compiler error since it enables `libstdc++` mode (on Xcode 10 at least - need to test when that error was dropped). This is largely for legacy build scripts.

Bumping it further that might just introduce further compile errors since it will start alerting for 10.9-10.14 deprecated API usage. If you feel strongly about it, I'd rather it was a separate PR so it's easier to revert.